nicegui/json/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
Is there a reason why this lives in an `__init__.py` file instead of a `json.py`?